src/linker/Mono.Linker.csproj

```diff
@@ -1,63 +1,38 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0">
-  <!--
-      TODO: Remove this workaround once supported.
+<Project Sdk="Microsoft.NET.Sdk">
 
-      The configuration isn't correctly set when the linker is
-      restored as a P2P reference due to a nuget bug
-      (https://github.com/NuGet/Home/issues/4873). As a workaround, we
-      check NuGetRestoreTargets, which is set by the .NET Core restore
-      targets.
-  -->
-  <PropertyGroup>
-    <ILLinkBuild Condition=" $(Configuration.StartsWith('illink')) Or '$(NuGetRestoreTargets)' != '' ">true</ILLinkBuild>
-    <ILLinkBuild Condition=" '$(ILLinkBuild)' == '' ">false</ILLinkBuild>
-    <TargetFrameworkProfile />
+  <PropertyGroup Condition=" $(Configuration.StartsWith('illink')) ">
+    <AssemblyName>illink</AssemblyName>
+    <Description>IL Linker</Description>
+    <DefineConstants>$(DefineConstants);FEATURE_ILLINK</DefineConstants>
+    <!-- net46 build is disabled until cecil uses SDK-style projects. -->
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(MSBuildRuntimeType)' == 'Core' And '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" ! $(Configuration.StartsWith('illink')) ">
+    <AssemblyName>monolinker</AssemblyName>
+    <AssemblyTitle>Mono.Linker</AssemblyTitle>
+    <Description>Mono CIL Linker</Description>
+    <TargetFrameworks>net462</TargetFrameworks>
   </PropertyGroup>
-  <Import Project="ILLink.props" Condition=" $(ILLinkBuild) " />
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.50727</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{DD28E2B1-057B-4B4D-A04D-B2EBD9E76E46}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <RootNamespace>Mono</RootNamespace>
-    <AssemblyName Condition=" ! $(ILLinkBuild) ">monolinker</AssemblyName>
-    <AssemblyName Condition=" $(ILLinkBuild) ">illink</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <Copyright>(C) 2006, Jb Evain</Copyright>
```

Comment:
Please use .NET Foundation for the copyright

Reply:
I'm happy to change the copyright info, but I'd rather do it in a separate PR. Here my goal is just to use the new SDK while reproducing the original build as closely as possible.